tests/x11/thunderbird/thunderbird_imap.pm

```diff
@@ -33,21 +34,29 @@ sub run {
 
     if ($hostname eq 'client') {
         $account = "internal_account_C";
+        # for imap  version, need disable screensaver/suspend
```

Comment:
you say imap version but then specify gnome. Meant to say "gnome"? Also a duplicate space after "imap"

Reply:
ok, done